opendebates/migrations/0024_auto_20160907_1024.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you delete these two migrations and re-create? It looks like there's been another change in `models.py` since these migrations were created (see the build failure), and we might as well just have a single migration to create the field the right way the first time.